src/dict.c

```diff
@@ -675,23 +694,36 @@ void dictRelease(dict *d)
 
 dictEntry *dictFind(dict *d, const void *key)
 {
-    dictEntry *he;
-    uint64_t h, idx, table;
+    dictEntry *he = NULL;
+    uint64_t h, idx;
 
     if (dictSize(d) == 0) return NULL; /* dict is empty */
-    if (dictIsRehashing(d)) _dictRehashStep(d);
```

Comment:
don't we wanna do that in the other flows (add and delete)?

Reply:
Yes we will be doing it in the other flows